.github/workflows/release.yml

```diff
@@ -0,0 +1,28 @@
+name: Release
+on:
+  push:
+    branches:
+      - main
+jobs:
+  release:
+    name: Create a GitHub Release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Get version from package.json
+        id: package-version
+        uses: martinbeentjes/npm-get-version-action@main
+      - name: Create a GitHub Release (Fails if release already exists with same version number)
+        uses: actions/github-script@v3
+        env:
+          package-version: ${{ steps.package-version.outputs.current-version }}
+        with:
+          github-token: ${{secrets.GITHUB_TOKEN}}
+          script: |
+            await github.repos.createRelease({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              tag_name: 'v' + process.env['package-version'],
+              generate_release_notes: true,
```

Comment:
Does the automatic generation of release notes base it on commit messages? Might be worth shouting out that our commit messages will be more visible and may need good titling...

Reply:
It uses the PR titles as far as we can tell.

Reply:
It will be links to PRs in the same way it works when we generate notes at the moment. Because we'll be updating the version each time we merge a PR pretty much, it will usually just link to that PR and have stuff like new contributors. So no need to worry about commit messages any more than usual!

Reply:
That sounds good, thanks both!